src/backend/gl/src/command.rs

```diff
@@ -109,6 +112,12 @@ struct Cache {
     program: Option<gl::types::GLuint>,
     // Blend per attachment.
     blend_targets: Option<Vec<Option<pso::ColorBlendDesc>>>,
+    // Maps bound vertex buffer offset (index) to handle.
+    vertex_buffers: Option<Vec<gl::types::GLuint>>,
```

Comment:
Is there a difference between an empty `Vec` and `None`?

Reply:
I don't think so, I don't see any advantage currently. I could simplify all of the `Option<Vec<_>>` fields if we don't need to distinguish between these.